test/images/agnhost/pause/pause.go

```diff
@@ -40,17 +40,17 @@ func pause(cmd *cobra.Command, args []string) {
 	done := make(chan int, 1)
 	signal.Notify(sigCh, syscall.SIGINT)
 	signal.Notify(sigCh, syscall.SIGTERM)
+	fmt.Println("Signals registered")
```

Comment:
... I'm not sure about our logging hygiene in agnhost but this seems comparable to existing code

@aojea @pohly

Reply:
Do we want to log this all the time? It looks more like debug output to me.

Reply:
I'm ok either way, but my reasoning is:
- this debug log line is useful to investigate failures of tests for external projects, like Kueue, due to race conditions. I imagine a popular issue to investigate will be a different exit code depending on the fact if the signal handlers were or were not registered. Such projects will rarely configure the image with increased verbosity.
- this is consistent with the first log line above saying `fmt.Println("Paused")`

Reply:
I don't mind, I was just wondering.